api/src/main/java/org/apache/cloudstack/api/command/user/network/CreateNetworkCmd.java

```diff
@@ -118,7 +118,8 @@ public class CreateNetworkCmd extends BaseCmd implements UserCmd {
     private Long projectId;
 
     @Parameter(name = ApiConstants.DOMAIN_ID, type = CommandType.UUID, entityType = DomainResponse.class, description = "domain ID of the account owning a network. " +
-            "If no account is provided then network will be assigned to the caller account and domain")
+            "If the account or acltype is not specified, the network will be automatically assigned to the caller account and domain." +
```

Comment:
if account is not specified, but acltype is Domain, the network will be created for the specified domain.